### PR TITLE
[release-4.13] Enable IT feature modal

### PR DESCRIPTION
**Pull request by avivtur**
## 📝 Description

Manual backport for #1330 

different links for 4.13 -
Developer Preveiw feature - `https://access.redhat.com/support/offerings/devpreview`
Learn more about instance types - `https://kubevirt.io/user-guide/virtual_machines/instancetypes/`
Quick start link removed as no quick start in 4.13 (we might add a link to a video on how to create VM from volume in a follow-up PR)

## 🎥 Demo

enable-it-413

### CODE REVIEW

File: locales/en/plugin__kubevirt-plugin.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
   "{{qualifiedNodesCount}} matching Nodes found": "{{qualifiedNodesCount}} matching Nodes found",
   "{{qualifiedNodesCount}} nodes found": "{{qualifiedNodesCount}} nodes found",
   "{{timestampPluralized}} ago": "{{timestampPluralized}} ago",
+  "<0><0><0>Creating VirtualMachine is easier than ever!</0></0><1><0>This <2>Developer Preview feature</2> provides a simple and quick way to create VirtualMachines. <6>You must have cluster admin permissions to enable this feature. You can disable it at any</6> <8>time on the Settings tab on the Overview page.</8></0></1></0>": "<0><0><0>Creating VirtualMachine is easier than ever!</0></0><1><0>This <2>Developer Preview feature</2> provides a simple and quick way to create VirtualMachines. <6>You must have cluster admin permissions to enable this feature. You can disable it at any</6> <8>time on the Settings tab on the Overview page.</8></0></1></0>",
```
Comment (apinnick) [inline locales/en/plugin__kubevirt-plugin.json:39]: Creating VirtualMachine > Creating a VirtualMachine

File: src/views/catalog/EnableInstanceTypeTechPreviewModal/components/EnableInstanceTypeContent.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import { Stack, StackItem, Text, TextVariants } from '@patternfly/react-core';
+
+const EnableInstanceTypeContent: FC = () => {
+  const { t } = useKubevirtTranslation();
+  return (
+    <Trans ns="plugin__kubevirt-plugin" t={t}>
+      <Stack hasGutter>
+        <StackItem>
+          <Text className="pf-u-font-size-xl pf-u-danger-color-100" component={TextVariants.p}>
+            Creating VirtualMachine is easier than ever!
```
Comment (apinnick) [inline src/views/catalog/EnableInstanceTypeTechPreviewModal/components/EnableInstanceTypeContent.tsx:14]: Creating VirtualMachine > Creating a VirtualMachine

File: locales/en/plugin__kubevirt-plugin.json (hunk 1 above)
Comment (avivtur) [reply locales/en/plugin__kubevirt-plugin.json]: @apinnick changed, great catch thanks!

File: src/views/catalog/EnableInstanceTypeTechPreviewModal/components/EnableInstanceTypeContent.tsx (hunk 2 above)
Comment (avivtur) [reply src/views/catalog/EnableInstanceTypeTechPreviewModal/components/EnableInstanceTypeContent.tsx]: @apinnick changed, thanks!

**Pull request merged**